k8s/upgrades/0.8.1-0.8.2/cstor_pool_upgrade.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+###########################################################################
+# STEP: Get SPC name and namespace where OpenEBS is deployed as arguments #
+#                                                                         #
+# NOTES: Obtain the pool deployments to perform upgrade operation         #
+###########################################################################
+
+pool_upgrade_version="v0.8.x-ci"
+current_version="0.8.1"
+
+function usage() {
+    echo
+    echo "Usage:"
+    echo
+    echo "$0 <spc-name> <openebs-namespace>"
+    echo
+    echo "  <spc-name> Get the SPC name using: kubectl get spc"
+    echo "  <openebs-namespace> Get the namespace where pool pods"
+    echo "    corresponding to SPC are deployed"
+    exit 1
+}
+
+##Checking the version of OpenEBS ####
+function verify_openebs_version() {
+    local resource=$1
+    local name_res=$2
+    local openebs_version=$(kubectl get $resource $name_res -n $ns \
+                 -o jsonpath="{.metadata.labels.openebs\.io/version}")
+
+    if [[ $openebs_version != $current_version ]] && [[ $openebs_version != $pool_upgrade_version ]]; then
+        echo "Expected version of $name_res in $resource is $current_version but got $openebs_version";exit 1;
+    fi
+    echo $openebs_version
+}
+
+## Starting point
+if [ "$#" -ne 2 ]; then
+    usage
+fi
+
+spc=$1
+ns=$2
+
+### Get the deployment pods which are in not running state that are related to provided spc ###
+pending_pods=$(kubectl get po -n $ns \
```

Comment:
we don't need these checks

Reply:
and the next one also that looks at pending_pods

Reply:
Done